modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase

```diff
@@ -26,6 +26,43 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test DownloadSampleFileOfTheCustomObject {
+		task ("Given a custom object is published") {
+			ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "student",
+				en_US_plural_label = "students",
+				name = "Student",
+				requiredStringFieldName = "name");
+		}
+
+		task ("And Given object entry is created") {
+			ObjectDefinitionAPI.createObjectEntryWithName(
+				en_US_plural_label = "students",
+				name = "Jessica");
+		}
+
+		task ("And Given I create an import file task with custom object as the entity type") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoImport();
+
+			ImportExport.selectEntity(entityType = "C_Student (v1_0 - Liferay Object REST)");
+		}
+
+		task ("When I download a sample file of this entity") {
+			Click.clickAt(locator1 = "ImportExport#SAMPLE_FILE_DOWNLOAD_BUTTON");
+		}
+
+		task ("Then the sample file download successfully") {
+			var sampleFileName = TestCase.getDownloadedTempFileName(fileNamePattern = "objectentry-*.csv");
+
```

Comment:
for this, I tried locally, and when you click the Download, the CSV file(objectentry#c_student-z592ypZ6.csv) will download, and no file named json_objectEntry_export.txt